portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase

```diff
@@ -4524,30 +4881,112 @@ definition {
+		task ("Create an experience, add a component and make changes without publish it") {
+			PageEditor.addFragment(
+				collectionName = "Layout Elements",
+				fragmentName = "Container");
 
+			Button.clickPublish();
+
+			PageEditor.addExperience(
+				experienceName = "New Experience",
+				segmentName = "Anyone");
+
+			Click(locator1 = "PageEditor#EXPERIENCE_LABEL");
+		}
+
+		task ("Go to the pages menu and discard changes") {
+			PageEditor.removeFragment(fragmentName = "Container");
+		}
+
+		task ("Go to Experience Content Page and the Experience is not present") {
+			PageEditor.viewFragmentNotPresent(fragmentName = "Container");
+
+			PageEditor.changeExperience(newExperience = "Default");
+
+			PageEditor.viewFragment(fragmentName = "Container");
+		}
 	}
 
 	@description = "LPS-86285: Validate in the content page if, the user can remove fragment in the experiences that can be different from others experiences (already published)"
-	@ignore = "true"
+	@ignore = "false"
 	@priority = "4"
 	test RemovingFragmentInDifferentExperiences {
+		task ("Go to Experience Content Page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Experience Content Page",
+				siteName = "Test Site Name");
 
-		// TODO LPS-86285 RemovingFragmentInDifferentExperiences pending implementation
+			PageEditor.addFragment(
+				collectionName = "Basic Components",
+				fragmentName = "Heading");
+		}
+
+		task ("Create an experience and add a fragment on both experiences") {
+			PageEditor.addExperience(
+				experienceName = "New Experience",
+				segmentName = "Anyone");
+
+			Button.clickPublish();
+		}
 
+		task ("Remove the fragment created from the default experience") {
+			PageEditor.removeFragment(fragmentName = "Heading");
+
+			Button.clickPublish();
+		}
+
+		task ("Check if the fragment is only present in the new experience") {
+			PageEditor.viewFragmentNotPresent(fragmentName = "Heading");
+
+			PageEditor.changeExperience(newExperience = "New Experience");
+
+			PageEditor.viewFragment(fragmentName = "Heading");
+		}
 	}
 
 	@description = "LPS-86285: Validate in the content page if, the user can remove fragment in the experiences that can be different from others experiences (not published)"
-	@ignore = "true"
+	@ignore = "false"
 	@priority = "3"
 	test RemovingFragmentOfUnpublishedExperiences {
+		task ("Go to Experience Content Page, add an experience and a fragment without publishing them") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Experience Content Page",
+				siteName = "Test Site Name");
+
+			PageEditor.addFragment(
+				collectionName = "Basic Components",
+				fragmentName = "Heading");
+
+			Button.clickPublish();
+
+			PageEditor.addExperience(
+				experienceName = "New Experience",
+				segmentName = "Anyone");
 
-		// TODO LPS-86285 RemovingFragmentOfUnpublishedExperiences pending implementation
+			Click(locator1 = "PageEditor#EXPERIENCE_LABEL");
```

Comment:
No need

Reply:
Same as explained here :

If I remove it the test fails with this error:
BUILD FAILED: Element is not present at "//div[contains(@Class,'cke_editable_inline') and contains(@id,'_content')] | //div[contains(@Class,'form-group')]//textarea[contains(@id,'_content')]"
[exec] /Users/robertstefan/Desktop/Liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro[editFragmentText]:2171
[exec] /Users/robertstefan/Desktop/Liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase[DiscardingChangesFromUnpublishedExperiences]:4159
I think is because when an Experience is added without clicking on Publish, the Selector of Experiences doesn't close so the test can't continue with the next function.

Reply:
You're right